venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,362 @@
+        "signatures":{"values-copied":["aclweb.org/ACL/2022/Conference"]},
+        "content":{
+            "paper_link": {
+                "description": "Provide the link to your previous ACL submission",
+                "value-regex": "http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\\(\\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+",
+                "required": True,
+                "order": 10
+                },
+            "paper_type": {
+                "description": "Select if your paper is short or long",
+                "value-radio": [
+                "short",
+                "long"
+                ],
+                "required": True,
+                "order": 11
+                },
+            "track": {
+                "description": "Select the track that best fits your submission.",
+                "values-checkbox": [
+                "Ethics and NLP",
+                "Linguistic theories, Cognitive Modeling and Psycholinguistics",
+                "Machine Learning for NLP",
+                "Phonology, Morphology and Word Segmentation",
+                "Resources and Evaluation",
+                "Semantics: Lexical",
+                "Semantics: Sentence-level Semantics, Textual Inference and Other areas",
+                "Syntax: Tagging, Chunking and Parsing",
+                "Information Extraction",
+                "Computational Social Science and Cultural Analytics",
+                "Information Retrieval and Text Mining",
+                "Interpretability and Analysis of Models for NLP",
+                "Machine Translation and Multilinguality",
+                "NLP Applications",
+                "Question Answering",
+                "Dialogue and Interactive Systems",
+                "Discourse and Pragmatics",
+                "Generation",
+                "Language Grounding to Vision, Robotics and Beyond",
+                "Sentiment Analysis, Stylistic Analysis, and Argument Mining",
+                "Speech and Multimodality",
+                "Summarization",
+                "Special Theme"
+                ],
+                "required": True,
+                "order": 12
+                },
+            "comment": {
+                "description": "Comment to Senior Area Chairs (500 words)",
+                "order": 13,
+                "value-regex": "[\\S\\s]{0,500}",
+                "required": False
+                }
+        }
+    }
+)
+client.post_invitation(commitment)
+
+# Creating SAC groups from Track Name 
+f = open('SAC_groups.csv')
+SAC_name_dictionary = {}
+
+file = csv.reader(f)
+header = []
+rows = []
+header = next(file)
+for row in file:
+    rows.append(row)
+f.close()
+for row in rows:
+    SAC_name_dictionary[row[1]] = row[0]
+
+
+
+# Retrieve all commitment submissions, ACL submissions, and ACL blind submissions  
+notes = openreview.tools.iterget_notes(client,invitation='aclweb.org/ACL/2022/Conference/-/Commitment_Submission')
```

Comment:
```suggestion
commitment_notes = openreview.tools.iterget_notes(client,invitation='aclweb.org/ACL/2022/Conference/-/Commitment_Submission')
```
let's use more descriptive names.